apps/docs/development/release.md

```diff
@@ -12,5 +12,7 @@ Currently the release process is manual.
     - packages/config/tailwind,
     - packages/sfui/frameworks/vue,
     - packages/sfui/frameworks/react.
-3. Create a PR with title: "chore(release): [NUMBER OF RELEASE]"
-4. After PR is merged tag it with `v[NUMBER OF RELEASE]` git tag.
+3. Make sure to update CHANGELOG.md with latest changes (for now manually).
+4. Create a PR with title: "chore(release): [NUMBER OF RELEASE]"
+5. After PR is merged tag it with `v[NUMBER OF RELEASE]` git tag.
+6. Create [Github release](https://github.com/vuestorefront/storefront-ui/releases) with information from `CHANGELOG.md` and referencing git tag you've just added.
```

Comment:
question: maybe it's worth to mention how to add Github Release https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

Reply:
8ab0dc3add4b62388faccdbe69bad4d5a2eb43b6